com.github.alainm23.planner.json

```diff
@@ -0,0 +1,94 @@
+{
+  "app-id": "com.github.alainm23.planner",
+  "sdk": "org.gnome.Sdk",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.34",
+  "base": "io.elementary.BaseApp",
+  "base-version": "juno-19.08",
+  "command": "com.github.alainm23.planner",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--device=dri",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--filesystem=home",
```

Comment:
What do you need full home access for?

Reply:
I need access to home because Planner saved his database in 
`/home /<user>/.local/share/com.github.alainm23.planner/...`

Reply:
How about using $XDG_DATA_HOME instead if set? Your app will have write access to that directory when run under Flatpak.

Reply:
You should use https://valadoc.org/glib-2.0/GLib.Environment.get_user_data_dir.html and it should work out of the box without home access.